lms/static/sass/lms-footer-rtl.scss

```diff
@@ -0,0 +1,24 @@
+// Footer for OpenEdX (right-to-left)
+// ==================================
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
```

Comment:
Nevermind, I'll get in and do a quick commit to clean that biz up. Stick to the big stuff :P